layers/+spacemacs/spacemacs-buffer-tabs/packages.el

```diff
@@ -0,0 +1,94 @@
+;;; packages.el --- tabs layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2018, 2020 Sylvain Benner & Contributors
+;;
+;; Author: Deepu Puthrote <git@deepumohan.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(defconst spacemacs-buffer-tabs-packages
+  '(centaur-tabs))
+
+(defun spacemacs-buffer-tabs/init-centaur-tabs ()
+  (use-package centaur-tabs
+    :defer t
+    :init
+    (progn
+      ;; configuration
+      (setq centaur-tabs-cycle-scope 'tabs
+            centaur-tabs-buffer-groups-function 'centaur-tabs-projectile-buffer-groups
+            centaur-tabs-set-icons t
+            centaur-tabs-gray-out-icons 'buffer
+            centaur-tabs-show-navigation-buttons nil
+            centaur-tabs-set-modified-marker t
+            centaur-tabs-modified-marker "⚠"
+            centaur-tabs-set-bar 'left
+            centaur-tabs-style 'bar)
+      ;; key bindings
+      (spacemacs/declare-prefix "[" "buffer tabs")
+      (spacemacs|spacebind
```

Comment:
This can be embedded into the `use-package` form like [here](https://github.com/syl20bnr/spacemacs/blob/develop/tests/layers/%2Bdistribution/spacemacs-base/spacebind-utest.el#L48)